tasks/main.yml

```diff
@@ -2,6 +2,11 @@
 # Include variables and define needed variables.
 - name: Include OS-specific variables.
   include_vars: "{{ ansible_os_family }}.yml"
+  when: ansible_os_family == 'Debian' or (ansible_os_family == 'RedHat' and ansible_distribution_major_version != '7')
```

Comment:
No need to include this `when`—including another vars file that overrides the same predefined var will fix it. I'll fix with a quick follow-up PR.